skymp5-server/cpp/server_guest_lib/MpObjectReference.cpp

```diff
@@ -296,8 +296,9 @@ void MpObjectReference::VisitProperties(const PropertiesVisitor& visitor,
     visitor("inventory", inventoryDump.data());
   }
 
-  if (mode == VisitPropertiesMode::All && !GetInventory().IsEmpty()) {
-    visitor("disabled", IsDisabled() ? "true" : "false");
+  const bool isEspmForm = GetFormId() < 0xff;
+  if (mode == VisitPropertiesMode::All && isEspmForm && IsDisabled()) {
```

Comment:
I think this is not needed `VisitPropertiesMode::All`. please check what it means

Reply:
There's the different mode is being set in case the emitter is the listener itself. ([click](https://github.com/skyrim-multiplayer/skymp/blob/main/skymp5-server/cpp/server_guest_lib/PartOne.cpp#L647) However, this is  the only place where I can find `VisitPropertiesMode::OnlyPublic`. So, it isn't checked somewhere, what makes me think this extra check is redundant. I can leave it there for consistency or remove if it is necessary.

Reply:
Mist be no `mode == VisitPropertiesMode::All`

otherwise won't be able to see neighbors values